src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs

```diff
@@ -26,23 +34,14 @@ public override void ConfigureServices(IServiceCollection services)
         public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
             var siteSettings = serviceProvider.GetService<ISiteService>().GetSiteSettingsAsync().GetAwaiter().GetResult();
+            var localizationSettings = siteSettings.As<LocalizationSettings>();
 
             var options = serviceProvider.GetService<IOptions<RequestLocalizationOptions>>().Value;
-
-            // If no specific default culture is defined, use the system language by not calling SetDefaultCulture
-            if (!String.IsNullOrEmpty(siteSettings.Culture))
-            {
-                options.SetDefaultCulture(siteSettings.Culture);
-            }
-
-            if (siteSettings.SupportedCultures.Length > 0)
-            {
-                var supportedCulture = siteSettings.GetConfiguredCultures();
-
-                options
-                    .AddSupportedCultures(supportedCulture)
-                    .AddSupportedUICultures(supportedCulture);
-            }
+            options.SetDefaultCulture(localizationSettings.DefaultCulture ?? "");
```

Comment:
When `DefaultCulture` will be `null`?!! while the intend is the default culture is always has a value